packages/react/src/patterns/blocks/FeatureCard/README.md

```diff
@@ -0,0 +1,94 @@
+# FeatureCard
+
+> The FeatureCard pattern is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/feature-card/index.scss';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { FeatureCard } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+const heading = 'Lorem ipsum dolor sit amet.';
+const card = [
+  {
+    heading: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+    copy:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+    cta: {
+      target: '_blank',
```

Comment:
nitpicky thing, but target isn't a minimum requirement, and thus I feel like it's not necessary (just adding noise) for the documentation.